Mathlib/Computability/Language.lean

```diff
@@ -268,15 +270,19 @@ instance : KleeneAlgebra (Language α) :=
       | zero => simp
       | succ n ih =>
         rw [pow_succ, mul_assoc (l^n) l m]
-        exact le_trans (le_mul_congr le_rfl h) ih,
+        exact le_trans (mul_le_mul_left' h _) ih,
     mul_kstar_le_self := fun l m h ↦ by
       rw [kstar_eq_iSup_pow, mul_iSup]
       refine iSup_le (fun n ↦ ?_)
       induction n with
       | zero => simp
       | succ n ih =>
         rw [pow_succ, ← mul_assoc m (l^n) l]
-        exact le_trans (le_mul_congr ih le_rfl) h }
+        exact le_trans (mul_le_mul_right' ih _) h }
+
+@[deprecated add_le_add (since := "2025-10-26")]
+theorem le_add_congr {l₁ l₂ m₁ m₂ : Language α} : l₁ ≤ m₁ → l₂ ≤ m₂ → l₁ + l₂ ≤ m₁ + m₂ :=
```

Comment:
Why did this get moved all the way down? Otherwise LGTM.

Reply:
The replacement for it isn't available until this point; it's implied by the instance above.